docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -146,3 +146,14 @@ Specifies minimal bucket to task ratio that has to be matched or exceeded in ord
 to use table scan node partitioning. When the table bucket count is small
 compared to the number of workers, then the table scan is distributed across
 all workers for improved parallelism.
+
+``colocated-joins-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+* **Session property:** ``colocated_join``
+
+*(Experimental)* Enables grouped join execution. If enabled, when both sides of
```

Comment:
`colocated-joins-enabled` is just that, it doesn't really enable `grouped execution`

Reply:
Updated, is that more technically accurate? Also, any feedback on the above discussion over the "Experimental" label?